src/record.ts

```diff
@@ -0,0 +1,124 @@
+type Category = {
+  duplicate: boolean;
+  nbTurns: number;
+  nbColors: number;
+  nbPossibilities: number;
+};
+
+export type Run = {
+  category: Category;
+  date: Date;
```

Comment:
We also want the nb of turn the player did before win (not always the number of turn max of the category)

Reply:
We could think about a better system to rank the run of the user based of the time took to finish the run and the lines took. Actually, it is just based on time.